app/src/main/java/me/guillaumin/android/osmtracker/db/DataHelper.java

```diff
@@ -65,6 +68,9 @@
 	 */
 	private Context context;
 
+	/** Statistics for all existing tracks */
+	private static TreeMap<Long, TrackStatistics> tracksStatistics = new TreeMap<Long, TrackStatistics> ();
```

Comment:
I'm not sure why you need to store something here? This class is supposed to be a stateless utility class so that doesn't sound right. Is that for caching purposes? In which cas that should be a field stored on the class that need this data, not this utility class.